.github/workflows/test.yml

```
@@ -40,6 +40,19 @@ jobs:
         id: fast-tests
         run: poetry run pytest --cov -m 'not slow'
 
+      - name: Get Chromium version 🌐
+        run: |
+          CHROMIUM_VERSION=$(wget -qO- https://googlechromelabs.github.io/chrome-for-testing/LATEST_RELEASE_STABLE | cut -d. -f1)
+          echo "Chromium version: $CHROMIUM_VERSION"
+          echo "CHROMIUM_VERSION=$CHROMIUM_VERSION" >> $GITHUB_ENV
+      - name: Setup Chrome 🌐
+        id: setup-chrome
+        uses: browser-actions/setup-chrome@v1
+        with:
+          chrome-version: ${{ env.CHROMIUM_VERSION }}
+      - name: Install chromedriver 🚗
+        run: |
+          npm install -g chromedriver@$CHROMIUM_VERSION
```

```suggestion
          npm install -g chromedriver@${CHROMIUM_VERSION}
```
personal preference: I like to make the `env` calls stand out a little more with the curly braces